p2p/peer.go

```diff
@@ -724,6 +719,16 @@ func (m *PeerManager) Dialed(peerID NodeID, address PeerAddress) error {
 
 	delete(m.dialing, peerID)
 
+	var upgradePeer NodeID
```

Comment:
This is the peer that we're gonna boot, i.e. upgrade from. Can we call this variable `upgradeFromPeer` just it's easier to read?